code/modules/SCP/SCPs/SCP-457.dm

```diff
@@ -0,0 +1,134 @@
+GLOBAL_LIST_EMPTY(scp457s)
+
+/datum/scp/scp_457
+	name = "SCP-457"
+	designation = "457"
+	classification = KETER
+
+/mob/living/scp_457
+	name = "SCP-457"
+	desc = "A horrible band of fire, it somehow keeps a humanoid shape."
+	icon = 'icons/SCP/scp-457.dmi'
+	icon_state = "fireguy"
+	SCP = /datum/scp/scp_457
+	status_flags = NO_ANTAG
+	var/door_cooldown
+	var/fuel = 30 //unused as time of coding
+
+	see_invisible = SEE_INVISIBLE_NOLIGHTING
+	see_in_dark = 7
+
+	maxHealth = 2000
+	health = 2000
+
+	can_pull_size = 3
+	a_intent = "harm"
+	can_be_buckled = FALSE
+
+	var/aflame_cooldown
+	var/aflame_cooldown_time = 2.8 SECONDS
+
+	var/area/spawn_area
+
+/mob/living/scp_457/Initialize()
+	GLOB.scp457s += src
+	spawn_area = get_area(src)
+	add_language(LANGUAGE_EAL, FALSE)
+	add_language(LANGUAGE_SKRELLIAN, FALSE)
+	add_language(LANGUAGE_GUTTER, FALSE)
+	add_language(LANGUAGE_SIGN, FALSE)
+	add_language(LANGUAGE_ENGLISH, TRUE)
+	set_light(0.8, 0.3, 5, l_color = COLOR_ORANGE) //makes 457 emit light
+	return ..()
+
+/mob/living/scp_457/Destroy()
+	GLOB.scp173s -= src
+	return ..()
+
+/mob/living/scp_457/UnarmedAttack(atom/A)
+	var/mob/living/carbon/human/H = A
+	if(ishuman(A))
```

Comment:
```suggestion
/mob/living/scp_457/UnarmedAttack(atom/A)
	if(ishuman(A))
		var/mob/living/carbon/human/H = A
```
nitpick, maybe, but still. Alternatively,
```suggestion
/mob/living/scp_457/UnarmedAttack(atom/A)
	var/mob/living/carbon/human/H = A
	if(istype(H))
```
the use of istype(<THING>) on a variable with a specific type (like `istype(H)` on `var/mob/living/carbon/human/H`) tells byond to check if the contents of that variable are of the type the variable expects (equivalent to `istype(H, /mob/living/carbon/human) basically, but shorter)`

Reply:
though keep in mind the latter of the two suggestions is more of a nitpick, but it looks nicer I guess?